### PR TITLE
Create /stats endpoint

### PR DESCRIPTION
**Pull request by tamminhdao**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 language: elixir
 elixir:
    - 1.6.3
-script: mix test
-
+script: MIX_ENV=test mix test
```
Comment (cmvandrevala) [inline .travis.yml]: I think in Travis you use the `env` header to set environment variables: https://docs.travis-ci.com/user/environment-variables/

File: lib/api_tictac_web/controllers/stats_controller.ex [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  def stat(conn, _params) do
+    call_stat = :ets.lookup(:stats_registry, :visits)
+                |> List.first()
+    number_of_visit = elem(call_stat, 1)
+    :ets.insert(:stats_registry, {:visits, number_of_visit + 1})
+
+    json conn, %{
+          "data" => [
+            %{"Page visited" => number_of_visit},
```
Comment (cmvandrevala) [inline lib/api_tictac_web/controllers/stats_controller.ex]: Maybe rename this to `number_of_visits` (notice the s at the end)

File: priv/repo/migrations/dummy_migration.ex [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Travis CI does not include empty folder
+# This file is here so that Travis CI can recognize this folder
```
Comment (cmvandrevala) [inline priv/repo/migrations/dummy_migration.ex:2]: If you have no migrations, why does your README tell you to run `mix ecto.migrate`? Can you get away with just `mix ecto.create`?

**Review by cmvandrevala — APPROVED** [review]

**Comment by cmvandrevala** [comment]
One other minor point about your README. I notice that you have each command on a separate line of a list with some description:

* Some description here `command 1`
* Some description here `command 2`

Instead try to format it like this:

Some description here. Some more description here.

```
command 1
command 2
```

This will allow the user to copy and paste easily

File: priv/repo/migrations/dummy_migration.ex (hunk 3 above)
Comment (tamminhdao) [reply priv/repo/migrations/dummy_migration.ex:2]: no, I tried that. doesnt work

Comment (cmvandrevala) [reply priv/repo/migrations/dummy_migration.ex:2]: Boo. That stinks. Ok, fair enough

Comment (tamminhdao) [reply priv/repo/migrations/dummy_migration.ex:2]: Sorry. I think I was not clear on this. 
The README was auto generated by Phoenix. I didn't write it myself. 
I didn't have to do `mix ecto.migrate`
But I meant, I tried to include `mix ecto.create` in the travis.yml file but it didn't work. I had to insert a dummy empty migration file in this folder for the build to pass.

**Pull request merged**